tools/gfx/vulkan/vk-shader-table.cpp

```diff
@@ -84,7 +83,7 @@ RefPtr<BufferResource> ShaderTableImpl::createDeviceBuffer(
 
         auto shaderGroupIndex = *shaderGroupIndexPtr;
         auto srcHandlePtr = handles.getBuffer() + shaderGroupIndex * handleSize;
-        memcpy(dstHandlePtr, srcHandlePtr, handleSize);
+        memcpy(dstHandlePtr, srcHandlePtr, rtProps.shaderGroupBaseAlignment);
```

Comment:
I think it should still be handleSize here.

Reply:
Also maybe good to set remaining bytes between habdleSize and alignment to 0.

Reply:
dstHandlePtr consistently shows up already zeroed out in the debugger for me, but done and done just to be safe.